tests/node/index.js

```diff
@@ -0,0 +1,32 @@
+var Fs = require('fs');
+var Path = require('path');
+var Env = require('../..').Env;
+var Fetch = require('../..').fetch;
+
+
+var env = new Env('en-US', Fetch);
+var ctx = env.createContext([
+  Path.join(__dirname + '/fixtures/{locale}/entities.l20n')
+]);
+var langs = [
+  {code: 'es-ES'},
+  {code: 'en-US'},
+  {code: 'fr'}
+];
+
+ctx.fetch(langs).then(init);
+
+function init () {
+  ctx.resolve(langs, 'whatCanIAsk').then(function (value) {
+    console.log('SINGLE VALUE: ', value);
+  });
+
+  Promise.all([
+    ctx.resolve(langs, 'whatCanIAsk'),
+    ctx.resolve(langs, 'openApp'),
+    ctx.resolve(langs, 'callNumber'),
+    ctx.resolve(langs, 'notFound')
+  ]).then(function (values) {
+    console.log('MULI VALUES', values);
+  });
```

Comment:
Another way of writing this code could be this:

```javascript
const identifiers = ['whatCanIAsk', 'openApp', 'callNumber', 'notFound'];
Promise.all(identifiers.map(ctx.resolve.bind(ctx, langs)));
```

..which I think is much simpler and also readable.

Regarding the return value of `Promise.all`, I don't think we can really return a dict because of interpolations.  You might want to interpolate different values into the same string, in which case a dict doesn't work.  Consider the following example:

```javascript
const strings = [
  ['aMsgFrom', 'Mary'],
  ['aMsgFrom', 'John']
];
Promise.all(strings.map(([id, args]) => ctx.resolve(langs, id, args)));
```

This also shows how the ES2015 syntax helps us write readable code with `Promise.all`.